### DISCUSSION TITLE
Why were versions 0.5.0 and 0.5.1 yanked?

### DISCUSSION THREAD

**Discussion by Velfi** [awslabs/aws-sdk-rust#422] [Q&A] [answered]
Why were versions 0.5.0 and 0.5.1 yanked from crates.io? Was it a security issue?

**Comment by rcoh** [comment, marked as answer]
0.5.0 and 0.5.1 were yanked to resolve a release issue where mismatching versions of the runtime crates (eg. aws-config et al) and the generated SDK crates (aws-sdk-ec2 et al) were erroneously released as 0.5.0 and 0.5.1. The release of 0.5.2 resolves this mismatch.
